src/main/resources/com/google/api/codegen/java/smoke_test.snip

```diff
@@ -10,21 +10,55 @@
     public static void main(String args[]) {
       Logger.getLogger("").setLevel(Level.WARNING);
       try {
-        executeNoCatch(args);
-        System.out.println("ok");
+        {@testBody(smokeTest)}
+        System.out.println("OK");
```

Comment:
Ummm in my opinion upper case "OK" is more eye-catching in the output so it would be more readable. But I am OK with both :8ball: